Google.Api.Generator.Tests/ProtoTests/Lro/Testing.Lro.GeneratedSnippets/snippet_metadata_testing.lro.json

```diff
@@ -1,272 +1 @@
-      "title": "ResourcedMethodAsync",
-      "description": "Snippet for ResourcedMethodAsync",
-      "file": "LroClient.ResourcedMethodAsyncSnippet.g.cs",
-      "language": "C_SHARP",
-      "clientMethod": {
-        "shortName": "ResourcedMethodAsync",
-        "fullName": "Testing.Lro.LroClient.ResourcedMethodAsync",
-        "async": true,
-        "resultType": "System.Threading.Tasks.Task<Google.LongRunning.Operation<Testing.Lro.LroResponse+Types+Nested, Testing.Lro.LroMetadata+Types+Nested>>",
-        "client": {
-          "shortName": "LroClient",
-          "fullName": "Testing.Lro.LroClient"
-        },
-        "method": {
-          "shortName": "ResourcedMethod",
-          "fullName": "testing.lro.Lro.ResourcedMethod",
-          "service": {
-            "shortName": "Lro",
-            "fullName": "testing.lro.Lro"
-          }
-        }
-      },
-      "origin": "API_DEFINITION"
-    },
-    {
-      "regionTag": "lro_generated_Lro_ResourcedMethod_sync_flattened_resourceNames",
-      "title": "ResourcedMethodResourceNames",
-      "description": "Snippet for ResourcedMethod",
-      "file": "LroClient.ResourcedMethodResourceNamesSnippet.g.cs",
-      "language": "C_SHARP",
-      "clientMethod": {
-        "shortName": "ResourcedMethod",
-        "fullName": "Testing.Lro.LroClient.ResourcedMethod",
-        "resultType": "Google.LongRunning.Operation<Testing.Lro.LroResponse+Types+Nested, Testing.Lro.LroMetadata+Types+Nested>",
-        "client": {
-          "shortName": "LroClient",
-          "fullName": "Testing.Lro.LroClient"
-        },
-        "method": {
-          "shortName": "ResourcedMethod",
-          "fullName": "testing.lro.Lro.ResourcedMethod",
-          "service": {
-            "shortName": "Lro",
-            "fullName": "testing.lro.Lro"
-          }
-        }
-      },
-      "origin": "API_DEFINITION"
-    },
-    {
-      "regionTag": "lro_generated_Lro_ResourcedMethod_async_flattened_resourceNames",
-      "title": "ResourcedMethodResourceNamesAsync",
-      "description": "Snippet for ResourcedMethodAsync",
-      "file": "LroClient.ResourcedMethodResourceNamesAsyncSnippet.g.cs",
-      "language": "C_SHARP",
-      "clientMethod": {
-        "shortName": "ResourcedMethodAsync",
-        "fullName": "Testing.Lro.LroClient.ResourcedMethodAsync",
-        "async": true,
-        "resultType": "System.Threading.Tasks.Task<Google.LongRunning.Operation<Testing.Lro.LroResponse+Types+Nested, Testing.Lro.LroMetadata+Types+Nested>>",
-        "client": {
-          "shortName": "LroClient",
-          "fullName": "Testing.Lro.LroClient"
-        },
-        "method": {
-          "shortName": "ResourcedMethod",
-          "fullName": "testing.lro.Lro.ResourcedMethod",
-          "service": {
-            "shortName": "Lro",
-            "fullName": "testing.lro.Lro"
-          }
-        }
-      },
-      "origin": "API_DEFINITION"
-    }
-  ]
-}
\ No newline at end of file
+﻿// TEST_DISABLE
```

Comment:
Snippet code is not being generated either here, so it really makes no sense to generate metadata, less so with line numbers in, etc.

LRO snippets are tested elsewhere, as well as the snippet metadata.